src/doc/contrib/src/architecture/codebase.md

```diff
@@ -3,106 +3,106 @@
 This is a very high-level overview of the Cargo codebase.
 
 * [`src/bin/cargo`](https://github.com/rust-lang/cargo/tree/master/src/bin/cargo)
-  — Cargo is split in a library and a binary. This is the binary side that
+  --- Cargo is split in a library and a binary. This is the binary side that
```

Comment:
This and other changes in this file shouldn't happen. They are markdown list items.

Reply:
> They are markdown list items.

No, they aren't. They are em dashes. May it be that your browser led you astray when you searched for hyphens on the page? This may also find em dashes (e.g., when the Firefox search-on-page bar option "Match Diacritics" is unchecked.

See also previous to last change in this file, where I changed the dash position from end of line to start of line for consistency.

It would also be strange if someone had used `*` *and* `-` for Markdown list items. And the deeply nested list structure would be strange.

Reply:
My bad. I didn't try to search `-` on my browser. Thank you for catching this 👍🏾.